core/tests/webdriverio_desktop/profileFeatures.js

```diff
@@ -233,6 +230,7 @@ describe('Playing the exploration', function() {
         await forms.toRichText('card 2'));
 
       // Test back button.
+      var backButton = $('.e2e-test-back-button');
```

Comment:
The issue is resolved now.

Reply:
Now resolved because the updated changes is leading to [flakiness.](https://github.com/oppia/oppia/runs/7092787230?check_suite_focus=true#step:12:16383)